docs/snippets/ov_extensions.cpp

```diff
@@ -40,6 +46,27 @@ class CustomOperation : public ov::op::Op {
     std::shared_ptr<ov::Node> clone_with_new_inputs(const ov::OutputVector&) const override { return nullptr; }
 };
 
+//! [frontend_extension_framework_map_macro_CustomOp]
+class CustomOp : public ov::op::Op {
+    std::string m_mode;
+    int m_axis;
+
+public:
+    OPENVINO_OP("CustomOp");
+    OPENVINO_FRAMEWORK_MAP(onnx, "CustomOp", { {"m_mode", "mode"} }, { {"m_axis", -1} });
```

Comment:
Why do you need to have new custom operation? Can you reuse operation from the `template_extension`?

Reply:
I could reuse Identity op from template_extension, but I also wanted to show how to map attributes.